### ISSUE TITLE
Lips don't sync on big faces.

### ISSUE THREAD

**Issue by Shariqaftab20** [vinthony/video-retalking#223]
I have a video on which I am trying to do lip syncing . the lips are generated well but when they are put back on to the original face , they don't align. I feel there me be two key reason . On e is in the input video the person is too expressive and opens his mouth too wide . second the face ratio to the rest of the frame is too high . what to do in such scenario ?. I am also uploading the input video and the generated video.

input video :  https://drive.google.com/file/d/1_b0-C9sKkSXlXMn1Yi6JZpeqHdzP_onE/view?usp=share_link
generated_ video:  https://drive.google.com/file/d/1VHaXd0ADsiLu7SzQzxpTQpqZfFNtC6aB/view?usp=share_link

**Comment by G-force78** [comment]
Yes the face is too close-up it was probably trained on ones that were not as large in the frame.
